### PR TITLE
Jetpack pricing page: update footer

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

This PR updates the footer of the Jetpack pricing page to match the new [jetpack.com](http://jetpack.com/) footer.

### Implementation notes

- Language toggle was not implemented in the previous footer. It'll be implemented here in a separate PR.
- The logic to hide the California notice for non Californian users will be implemented in a later PR too.

### Testing instructions

- Download this PR and run cloud
- Visit `/pricing/`
- Check that the footer is similar to the Jetpack.com footer, keeping in mind the two points above
- Check that each link directs you to the page it advertises
- Check that tracking events are triggered when clicking a link

### Screenshots
Screen Shot 2022-02-02 at 2 21 41 PM

### CODE REVIEW

File: client/my-sites/plans/jetpack-plans/selector.tsx [hunk 1]
```diff
@@ -218,9 +218,8 @@ const SelectorPage: React.FC< SelectorPageProps > = ( {
 				{ siteId && <QuerySiteProducts siteId={ siteId } /> }
 				{ siteId && <QuerySitePurchases siteId={ siteId } /> }
 				{ siteId && <QuerySites siteId={ siteId } /> }
-
-				{ footer }
 			</Main>
+			{ footer }
```
Comment (monsieur-z) [inline client/my-sites/plans/jetpack-plans/selector.tsx:225]: From an accessibility standpoint, footer should be outside of the `main` tag.

**Review by manzoorwanijk — APPROVED** [review]
Tested both on Desktop and Mobile. Everything looks great and as expected ✔️

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7124622

Thank you @monsieur-z for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7124622).

**Pull request merged**